pkg/config/toml/source.go

```diff
@@ -33,3 +33,16 @@ func FromFile(path string) Loader {
 	}
 	return tomlFile(path)
 }
+
+// FromCommandLine creates a TOML source from the output
+// of a shell command specified via string slice.
+// If an empty slice is passed an empty toml config is used.
+func FromCommandLine(params []string) Loader {
```

Comment:
Is there any reason not to use:
```suggestion
func FromCommandLine(cmd string, args ...string) Loader {
```
here instead?